### PR TITLE
UI changes to Jetpack Scan for Codeable partnership

### PR DESCRIPTION
**Pull request by scottsweb**
#### Changes proposed in this Pull Request

Forked from previous PR https://github.com/Automattic/wp-calypso/pull/47274, looking to merge that one first. 

This PR introduces some UI refinements to the Jetpack Scan product to support a new partnership with Codeable. Some context can be found here:

p1HpG7-aAg-p2
p1HpG7-ayK-p2

**For copy review:**

We have two new paragraphs for security threats that cannot be fixed automatically, they are:

> Jetpack Scan cannot automatically fix this threat. Consider resolving the threat manually. Check WordPress, your theme, and plugins are up to date. You may also consider removing the offending code, plugin or theme from your site.

> For further help, we recommend our partners Codeable. Their pricing ranges from $70-120/hour, with no obligation to hire. Codeable is a WordPress-exclusive freelancer marketplace with a community of 530+ highly vetted security experts. Get a free, no obligation estimate for resolving this threat.

Some of this was supplied to us, but would love your thoughts on how we might better structure the sentences. 

**For code review:**

Please check that the external link is being properly tracked and logged. I would also like a sanity check on some of the other changes, like being able to support two paragraphs in the threat description.

**For design review:**

I have tweaked the paragraph spacing and made some other layout changes around the new buttons and copy. Please check they look as expected. Please also check in Calypso blue: https://calypso.live/?branch=new/jetpack-scan-codeable

Before:

cloud jetpack com_backup_meaningful-ape jurassic ninja_site=scottsweb wordpress com

After:

jetpack cloud localhost_3000_scan_meaningful-ape jurassic ninja

hash-b92983cef842ab85a509eb40adede489203c5ee1 calypso live_scan_meaningful-ape jurassic ninja

#### Testing instructions

* Grab this PR
* Run `npm start start-jetpack-cloud`
* Visit `http://jetpack.cloud.localhost:3000/`
* Make sure you have a Jetpack site with some security threats
* Check the new page as per above, also check the history of threats to make sure there are no breaking changes

### CODE REVIEW

File: client/components/jetpack/threat-description/index.tsx [hunk 1]
```diff
@@ -24,11 +24,34 @@ export interface Props {
...
+		const { status, isFixable } = this.props;
+
+		switch ( status ) {
+			case 'fixed':
+				return translate( 'How did Jetpack fix it?' );
+				break;
+
+			case 'current':
+				if ( isFixable ) {
+					return translate( 'How will we fix it?' );
```
Comment (a8ci18n) [inline client/components/jetpack/threat-description/index.tsx:45]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'How we will fix it?' )` *ES Score: 7*

File: client/my-sites/scan/main.tsx [hunk 2]
```diff
@@ -185,7 +185,7 @@ class ScanPage extends Component< Props > {
 				) }
 				<p>
 					{ translate(
-						'We will send you an email once the scan completes, in the meantime feel ' +
+						'We will send you an email if security threats are found. In the meantime feel ' +
```
Comment (a8ci18n) [inline client/my-sites/scan/main.tsx:188]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'We will send you an email once the scan completes, in the meantime feel free to continue to use your site as normal, you can check back on progress at any time.' )` *ES Score: 6*

File: client/components/jetpack/scan-threats/index.tsx [hunk 3]
```diff
@@ -219,16 +240,21 @@ const ScanThreats = ( { error, site, threats }: Props ) => {
...
-							components: {
-								button: (
-									<Button className="scan-threats__run-scan-button" onClick={ dispatchScanRun } />
-								),
-							},
-						}
-					) }
+					<p className="scan-threats__rerun-help">
+						{ translate(
+							'If you have manually fixed any of the threats above, you can {{button}}run a scan now{{/button}} or wait for Jetpack to scan your site later today.',
```
Comment (a8ci18n) [inline client/components/jetpack/scan-threats/index.tsx:245]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'If you have manually fixed any of the threats listed above, you can {{button}}run a manual scan now{{/button}} or wait for Jetpack to scan your site later today.' )` *ES Score: 9*

File: client/components/jetpack/threat-item-subheader/index.tsx [hunk 4]
```diff
@@ -66,11 +67,21 @@ const getThreatStatusMessage = ( translate, threat: Threat ) => {
 	return null;
 };
 
+const getAutoFixBadge = ( translate, threat: Threat, isFixable ) => {
+	if ( isFixable && threat.status !== 'fixed' ) {
+		return (
+			<Badge className={ classnames( 'threat-item-subheader__badge', 'is-auto-fix' ) }>
+				<small>{ translate( 'Auto fix' ) }</small>
```
Comment (a8ci18n) [inline client/components/jetpack/threat-item-subheader/index.tsx:74]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Auto Fixes' )` *ES Score: 7*

File: client/components/jetpack/scan-threats/index.tsx [hunk 5]
```diff
@@ -161,47 +161,61 @@ const ScanThreats = ( { error, site, threats }: Props ) => {
 			<h1 className="scan-threats scan__header">{ translate( 'Your site may be at risk' ) }</h1>
 			<p>
 				{ translate(
-					'The scan found {{strong}}%(threatCount)s{{/strong}} potential threat with {{strong}}%(siteName)s{{/strong}}.',
-					'The scan found {{strong}}%(threatCount)s{{/strong}} potential threats with {{strong}}%(siteName)s{{/strong}}.',
+					'Jetpack Scan found {{strong}}%(threatCount)s{{/strong}} potential threat on {{strong}}%(siteName)s{{/strong}}. Please review the threat and take action.',
```
Comment (a8ci18n) [inline client/components/jetpack/scan-threats/index.tsx:164]: :warning: This change will be queued for retranslation. We'll display the English text until then.

**Comment by folletto** [comment]
It seems matching the i3 of the designs, so I think it's all good under that regard.

**Comment by clucasrowlands** [comment]
Regarding the copy, here's my suggested edit:

> Jetpack Scan cannot automatically fix this threat. We suggest that you resolve the threat manually: ensure that WordPress, your theme, and all of your plugins are up to date, and remove the offending code, plugin, or theme from your site.
> 
> If you need more help on resolving this threat, we recommend Codeable, a WordPress freelancer marketplace of 530+ highly vetted security experts. Pricing ranges from $70-120/hour, and you can get a free estimate with no obligation to hire.

The original phrasing could be more direct and succinct (eg, "consider resolving," "consider removing"). It also sounds like fixing the threat is not mandatory or necessary. Regardless of whether or not that is accurate, it'd still be better to tighten up the copy if possible.

For the second graf, I removed "partners" when referring to Codeable. If this refers to an official partnership we have with the platform itself, you can tweak that sentence to:

> If you need more help on resolving this threat, we recommend our partner Codeable, which is a WordPress freelancer marketplace of 530+ highly vetted security experts. 

Otherwise, the copy above should work!

**Review by rcanepa — APPROVED** [review]
Your changes look good to me, @scottsweb. I also made sure to test that the external link is being properly tracked and logged. 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5222782

Thank you @scottsweb for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5222782).

**Pull request merged**